### DISCUSSION TITLE
Install Location

### DISCUSSION THREAD

**Discussion by ianalexis** [Devolutions/UniGetUI#829] [General]
Now all apps install in some default folder (some Winget apps miss the shortcut and i cant find where has been installed, but reinstalling them by PowerShell sometimes works).
I think something like a CRUD list in settings to add directorys and select it when installing the package (like steam).
This way, for example, i can avoid to fill up my NVME with little apps and just use any other HDD for it.

**Winget**: uses `--location` and `-i` as install arguments to do it (idk the difference between them).
**Chocolatey**: I couldn't find (less than 10m google search) this option.
**Scoop**: Maybe changing the Environment Variable path but this affect alll Scoop Packages.

**Comment by de-Jo** [comment]
I would highly endorse that request.
It would be really important to be able to choose a the installation path / directory.

**Comment by Spacellary** [comment]
Maybe a custom argument button with a few preset/configurable and custom options would work.

**Comment by marticliment** [comment]
Hello, 

This feature is already available on version 2.2.0:
![image](https://github.com/marticliment/WingetUI/assets/53119851/22aedecf-4d37-42bd-acda-56429a8fa9ae)

**Reply by Spacellary** [reply]
Marti being 3 parallel universes ahead of me.
